src/qt/locale/bitcoin_pt_PT.ts

```diff
@@ -21,25 +19,20 @@ This is experimental software.
 Distributed under the MIT/X11 software license, see the accompanying file COPYING or https://opensource.org/licenses/mit-license.php.
 
 This product includes software developed by the OpenSSL Project for use in the OpenSSL Toolkit (https://www.openssl.org/) and cryptographic software written by Eric Young (eay@cryptsoft.com) and UPnP software written by Thomas Bernard.</source>
-        <translation type="unfinished"></translation>
+        <translation>
+Este é um software experimental-
+
+Distribuído através da licença MIT/X11, veja o ficheiro que acompanha ou em https://opensource.org/licenses/mit-license.php..
```

Comment:
Nitpicks, but why did it change the period to a dash, and then also the double period after mit-license.php.

Reply:
my mistake, already fixed